src/components/tile/Tile.js

```diff
@@ -1,18 +1,35 @@
 import React from "react";
```

Comment:
You could seperate this into two components too similar to your removeContact function. One for the `appointmentTile` and one for the `contactTile`. For now its fine the way it is, but if you intended to extend either contacts or appointments, you end up needing to add heaps of:
if contact {do something specific for contacts}  else {do something specific for appointments}
This can make these components big and messy down the line.